conans/client/api/conan_api.py

```diff
@@ -239,39 +239,47 @@ def search_remote_recipes(self, query, remote):
         data["results"] = results
         return data
 
-    @api_method
-    def get_remote_recipe_revisions(self, reference, remote):
-        ref = ConanFileReference.loads(reference)
+    def _get_revisions(self, reference, ref_type, remote=None):
+        if ref_type == "package":
```

Comment:
This type conversion is better done outside the ``_get_revisions``, then you avoid the ``ref_type`` argument, which would be redundant (you use the ``reference`` argument type, more idiomatic)

Reply:
This is correct, but you also need `ref_type` for knowing which remote method you are calling (line 249).

You could do that by checking the type of `reference`, though. It becomes a little messy, but you're right that from a UX perspective, `ref_type` should be implicit.

Reply:
``ref_type`` should be automatically deduced from the ``reference`` type, I agree is not perfect, but still better than having to pass ``"package"`` string explicitly as an argument

Reply:
Much better 👍